examples/e14_slash_commands/src/main.rs

```diff
@@ -77,21 +77,25 @@ impl EventHandler for Handler {
         let c1 = CreateCommand::new("ping", "A ping command");
 
         let c2 = CreateCommand::new("id", "Get a user id").add_option(
-            CreateOption::new(CommandOptionType::User, "id", "The user to lookup").required(true),
+            CreateOption::new(CommandOptionType::User, "id")
```

Comment:
Shouldn't this change be applied on `CreateApplicationCommand` instead? I believe descriptions being required for options should be fine.

Reply:
:facepalm: you're right. I got the two builders mixed up. The actual fix is now pushed :upside_down_face: